entities/longline.ts

```diff
@@ -0,0 +1,18 @@
+export enum LonglineType {
+  SmallBuoy = "small-buoy", // can have 280 containers
+  BigBuoy = "big-buoy", // can have 200 containers
+}
+
+export enum AcceptedContainerTypes {
+  Seed = "seed",
+  Shell = "shell", // juvenile oyster
+  Baby = "baby",
+  Medium = "medium",
+  Large = "large",
+}
+
+export type LongLine = {
+  position_at_farm: number;
```

Comment:
Aqui era um array não era? Pra casos de farms grandes

```suggestion
  position_at_farm: number[];
```